packages/budget/package.json

```diff
@@ -29,7 +29,7 @@
     "start:dev": "cross-env NODE_ENV=development node server",
     "start:prod": "cross-env NODE_ENV=production node server",
     "test": "cross-env BABEL_ENV=test mocha --opts ./mocha.opts 'src/**/*.test.js'",
-    "lint": "eslint --fix src"
+    "lint": "eslint src"
```

Comment:
I'm getting this error when running the command in this package:
```bash
dna budget: (lint-scripts)$ yarn run lint
yarn run v1.13.0
$ eslint src

Oops! Something went wrong! :(

ESLint: 5.16.0.
You are linting "src", but all of the files matching the glob pattern "src" are ignored.

If you don't want to lint these files, remove the pattern "src" from the list of arguments passed to ESLint.

If you do want to lint these files, try the following solutions:

* Check your .eslintignore file, or the eslintIgnore property in package.json, to ensure that the files are not configured to be ignored.
* Explicitly list the files from this glob that you'd like to lint on the command-line, rather than providing a glob as an argument.

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
Ah, there's a .eslintignore file in that package that ignores ... `**/*.js`!

I wonder if there's a command eslint that will turn no-files into a pass. That way, we could enforce linting for the whole project at the CI level for all of the packages that have been linted, before all of the packages have been linted.

Reply:
It looks like there's not a command to pass to eslint for this, but they're working on it (eslint/eslint#10587)

We could workaround, if we feel like going for it. A couple of reference examples:
- > We're currently shipping a file called eslint-no-cry.js in almost every seed, which have the only purpose of avoiding eslint cli to exit the build process when there is no .js file to be linted (eslint/eslint#10587).
- https://github.com/davidchambers/doctest/pull/111/files

Reply:
Nifty. Do you want to do that or shall I add the commit?

Reply:
All you :)

Reply:
Fixed! Since the local `.eslintignore` is used I _think_ we're good for a travis build, but just in case something goes wrong in building we can check out the `.travis.yml`.

Reply:
After you review my change I'll aprove